examples/example-add-feedstock-output.yml

```diff
@@ -1,6 +1,4 @@
 action: add_feedstock_output
 feedstock_to_output_mapping:
   # this entry adds a package name
-  - clang-compiler-activation: clang_impl_osx-64
-  # this entry adds an allowed glob pattern
-  - llvmdev: "libllvm*"
+  - sqlite-zstd: sqlite-zstd-python
```

Comment:
You are modifying the example here. You should create a new request file based on it instead. See https://github.com/conda-forge/admin-requests/pull/1228/files for a PR example.

Reply:
🤦 Thanks, should be fixed now.